<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLCamera</class>
 <widget class="QWidget" name="WLCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCameraViewfinder" name="viewCamera" native="true">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbResolution"/>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="cbCamera"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
